backend/core/src/Core.API/appsettings.json

```diff
@@ -69,6 +69,7 @@
     "AllowedOrigins": []
   },
   "Quartz": {
+    "ProcessEmailsJob": "0 * * * * ?",
```

Comment:
Should the sendgridoptions also be added to the appsettings?

Reply:
oh yes, I thought I have them here

Reply:
Done